src/power.rs

```diff
@@ -18,6 +19,17 @@ pub enum PowerMode {
     UltraLowPower(LowPowerMode),
 }
 
+pub enum WakeUpLine {
+    Internal,
+    WKUP1,
+    WKUP2,
+    // TODO: missing in SVD file
+    // WKUP3,
+    WKUP4,
+    WKUP5,
+    WKUP6,
```

Comment:
Does it make sense to use rust idiomatic `Wkup1` or it probably makes it actually more confusing for the user.

Reply:
Good point!
I was using abbreviation from datasheet, but yes, you right, we must use more idiomatic and self-descriptive names.